util/fileglob.go

```diff
@@ -37,3 +49,48 @@ func ListFilesGlob(ctx context.Context, base string, pattern string) []string {
 	}
 	return list
 }
+
+// Determines if a file path contains a hidden file or directory.
+// Either it starts with . or or contains '/.' to be considered hidden.
+func isHidden(fp string) bool {
+	return strings.HasPrefix(fp, ".") || strings.Contains(fp, "/.")
+}
+
+// ListFilesInVersion is the new, optimized version of ListFilesGlob created for
+// package generation. It lists all of the files within a particular cdnjs package version in
+// the same manner as ListFilesGlob with a '**' glob pattern.
+// Note that hidden cdnjs versions and hidden files/directories are ignored.
+// It utilizes the fast godirwalk library found here: https://github.com/karrick/godirwalk
+func ListFilesInVersion(ctx context.Context, base string) []string {
+	list := make([]string, 0)
+
+	// check if the version is hidden
+	if isHidden(base) {
+		Debugf(ctx, "ignoring hidden version %s", base)
+		return list
+	}
+
+	// walk the files recursively within the cdnjs package version directory
+	err := godirwalk.Walk(base, &godirwalk.Options{
+		Callback: func(fp string, de *godirwalk.Dirent) error {
+			// trim a full path to a path relative to the base directory (inside package version dir)
+			// trim any leading '/' for consistency with legacy ListFilesGlob implementation
+			fp = strings.TrimLeft(strings.TrimPrefix(fp, base), "/")
+			// path must not be a directory, not be hidden, and not be empty
+			if !de.IsDir() && !isHidden(fp) && fp != "" {
+				list = append(list, fp)
+			}
+			return nil
+		},
+		FollowSymbolicLinks: true,
+	})
+
+	// if non-nil error, attempt to try with the legacy version
+	// for example, at least one package (history) has
+	// symlink js files that link to themselves due to case
+	// insensitivity on some operating systems, causing an error
+	if err != nil {
```

Comment:
can you log the error here? util.Debugf

Reply:
Since this should not happen on linux where it is running, I just commented it all out and replaced with Check(err)